runtime/doc/api.txt

```diff
@@ -646,7 +646,7 @@ nvim_call_atomic({calls})                                 *nvim_call_atomic()*
                  arguments.
 
     Return: ~
-        Array of two elements. The first is an array of return values. The
+        An array of two elements. The first is an array of return values. The
```

Comment:
Do not do add articles everywhere. This is explicitly called out in `:help dev-doc`.